# This message is for debug purposes.
# It share the controller internal state
builtin_interfaces/Time stamp

float64 max_voltage
float64 max_turnrate
float64 max_acceleration

float64 heading_ref
float64 turn_rate
float64 vel_dif
float64 acc_coeff

float64 vright_ref
float64 aright_ref
float64 volt_right
float64 vleft_ref
float64 aleft_ref
float64 volt_left

bool acc_saturation
bool voltage_saturation
bool turnrate_saturation
